matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -160,26 +161,29 @@ async fn message_loop<M: Messenger>(
 
         select! {
             _  = &mut timeout => {
-                requests_sender.unbounded_send(PeerRequest::KeepAlive).expect("send failed");
-                // UNWRAP: we will only ever get here if there already was a timeout
-                let interval = keep_alive_interval.unwrap();
-                timeout = Either::Left(Delay::new(interval)).fuse();
+                requests_sender.unbounded_send(PeerRequest::KeepAlive).map_err(TrySendError::into_send_error)?;
+                if let Some(interval) = keep_alive_interval {
+                    timeout = Either::Left(Delay::new(interval)).fuse();
+                }
```

Comment:
If we remove the `.unwrap`, could we please add an `error` so it's clear we always intend this to be the case?

Reply:
Where is the unwrap?

If the KeepAlive fails to send, it exits the message loop with `Err` now.

Reply:
What I mean is that `if let Some(interval) = keep_alive_interval` will always be the case.

The old code communicated that this would always be the case

```rust
// UNWRAP: we will only ever get here if there already was a timeout
let interval = keep_alive_interval.unwrap();
```

In the new code it looks like this is something that can not be the case. I'd like us to know if the timeout is not restarted properly